docs/src/dataDisplay/chip/examples/exampleChip.jsx

```diff
@@ -0,0 +1,39 @@
+import {
+    Chip,
+} from 'react-cm-ui';
+import makeStyles from 'react-cm-ui/styles/makeStyles';
+import React from 'react';
+
+const useStyles = makeStyles(({
+    spacing,
+}) => ({
+    root: {
+        display: 'flex',
+        justifyContent: 'center',
+        flexWrap: 'wrap',
+        '& > *': {
+            margin: spacing(0.5),
+        },
+    },
+}));
+
+function ExampleChip() {
+    const classes = useStyles();
+
+    return (
+        <div className={classes.root}>
+            <Chip color="green" disabled label="Disabled" />
+            <Chip color="green" label="Green" />
+            <Chip color="teal" label="Teal" />
+            <Chip color="sky" label="Sky" />
+            <Chip color="teal" label="Teal" />
```

Comment:
Teal is in here twice?  Is there another color we are supposed to include?

Reply:
Fixed.

Reply:
Ah ha!  _Cyan!_
Resolving.